### PR TITLE
Add new apis for managing jobs.

### PR DESCRIPTION
**Pull request by geekbeast**

### CODE REVIEW

**Review by makosblade — APPROVED** [review]

**Review by katherinebernstein — APPROVED** [review]

File: src/main/kotlin/com/openlattice/admin/AdminController.kt [hunk 1]
```diff
@@ -223,6 +224,22 @@ class AdminController : AdminApi, AuthorizingComponent {
         return jobService.getJobs(listOf(jobId))
     }
 
+    @Timed
+    @PatchMapping(
+            value = [JOBS + ID_PATH],
+            produces = [MediaType.APPLICATION_JSON_VALUE],
+            consumes = [MediaType.APPLICATION_JSON_VALUE]
+    )
+    override fun updateJob(jobId: UUID, update: JobUpdate): Map<UUID, AbstractDistributedJob<*, *>> {
```
Comment (UnsungHero97) [inline src/main/kotlin/com/openlattice/admin/AdminController.kt]: Shouldn't there be `@PathVariable` annotations here?

```
fun updateJob(@PathVariable(ID) jobId: UUID, @PathVariable(UPDATE) update: JobUpdate)
```

Comment (geekbeast) [reply src/main/kotlin/com/openlattice/admin/AdminController.kt]: Yup, I was pushing up some incomplete local work so I could generate more incomplete local work >_<

Comment (geekbeast) [reply src/main/kotlin/com/openlattice/admin/AdminController.kt]: Also the second one should be ```@RequestBody``` not pathvariable.

**Comment by geekbeast** [comment]
Overriding in spite of build, because of multiple API changes colliding.

**Pull request merged**